tests/D2L.CodeStyle.Analyzers.Test/Specs/AttributeAliasesAnalyzer.cs

```diff
@@ -0,0 +1,59 @@
+﻿// analyzer: D2L.CodeStyle.Analyzers.Language.AttributeAliasesAnalyzer
+
+using System;
+
+using RootAliasAttribute = Test.Attributes.TestAttribute;

```

Comment:
Probably a bit silly but a test case for `using static` to pull an attribute out of a static class? Ahem `Objects.Immutable` 😒

Reply:
Added